spec/components/layout/footer_component_spec.rb

```diff
@@ -20,4 +20,26 @@
 
     expect(page).not_to be_rendered
   end
+
+  describe "when the cookies consent feature is enabled" do
```

Comment:
Note we're using a describe block containing two tests, and one of those tests does the opposite of what the describe block says :thinking:.

```suggestion
  describe "link to manage cookies" do
```